_board/hxr_sao_dmm.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: download
+board_id: "hxr_sao_dmm"
+title: "SAO Digital Multimeter Download"
+name: "SAO Digital Multimeter"
+manufacturer: "HXR.DK"
+board_url:
+ - "https://github.com/flummer/dmm-sao"
+board_image: "hxr_sao_dmm.jpg"
+date_added: 2025-02-24
+family: rp2040
+downloads_display: true
+blinka: false
+download_instructions: ""
```

Comment:
Maybe stick in a link to the paragraph above the flashing instructions, where it mentions using a custom build of circuitpython (to get the needed libraries)

I'd probably link to:
https://github.com/flummer/dmm-sao/?tab=readme-ov-file#firmware